src/index.html

```diff
@@ -0,0 +1,482 @@
+            <div class="bread__crust"></div>
+          </div>
+        </li>
+        <li class="header__item">
+          <img
+            class="header__icon"
+            src="css/images/icon/apple_image__small.svg"
+            alt="apple-logo"
+          />
+        </li>
+        <li class="header__item">
+          <img
+            class="header__icon"
+            src="css/images/icon/bag_image__small.svg"
+            alt="bag"
+          />
+        </li>
+      </ul>
+      <!-- Header for Tablet, Laptop -->
+      <ul class="header header--only-visible-tablet">
+        <li class="header__item">
+          <img
+            class="header__icon"
+            src="css/images/icon/apple_image__small.svg"
+            alt="apple-logo"
+          />
+        </li>
+        <li class="header__item">Store</li>
+        <li class="header__item">Mac</li>
+        <li class="header__item">iPad</li>
+        <li class="header__item">iPhone</li>
+        <li class="header__item">Watch</li>
+        <li class="header__item">TV</li>
+        <li class="header__item">Music</li>
+        <li class="header__item">Support</li>
+        <li class="search-item header__item">
+          <div class="header__icon"></div>
+        </li>
+        <li class="header__item">
+          <img
+            class="header__icon"
+            src="css/images/icon/bag_image__small.svg"
+            alt="bag"
+          />
+        </li>
+      </ul>
+    </header>
+
+    <!-- Hero -->
+    <section class="card hero-card hero-background">
+      <div class="card__container">
+        <div class="card__content content hero-content content--theme_dark">
+          <h2 class="content__headline">Now for the highlights.</h2>
+          <h3 class="content__subhead">
+            Introducing iPhone 13 Pro, iPhone 13, Apple Watch Series 7, and the
+            new iPad mini and iPad.
+          </h3>
+          <div class="content__cta-links">
+            <span class="content__cta-link">Watch the event</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- iPhone 13 Pro -->
+    <section class="card iphone-13-pro-card">
+      <div class="card__container">
+        <div
+          class="
+            card__content
+            iphone-13-pro-content
+            content content--theme_light
+          "
+        >
+          <h2 class="content__headline">iPhone 13 Pro</h2>
+          <h3 class="content__subhead">
+            A dramatically more powerful camera system. An all-new OLED display
+            with ProMotion. The world's fastest smartphone chip. A huge leap in
+            battery life.
+          </h3>
+          <div class="content__cta-links">
+            <span class="content__cta-link">Learn more</span>
+            <span class="content__cta-link">View pricing</span>
+          </div>
+        </div>
+        <div class="card__image-wrapper">
+          <div class="card__image"></div>
+        </div>
```

Comment:
div 에 대한 background 로 이미지를 처리하면 img 태그의 alt 대체 텍스트 속성을 대체할 다른 수단이 필요할 듯 보입니다!

Reply:
딱히 좋은 방법이 생각나질 않네요..  :sob: 
card__image 를 figure 요소로 변경하고, aria-label 속성을 추가해 보았습니다. 
혹시 다른 좋은 방법을 알고 계신다면 공유해주시면 감사하겠습니다 :dog: 

```suggestion
        <div class="card__image-wrapper">
          <figure class="card__image" aria-label="the back of iPhone 13 pro"></figure>
        </div>
```

Reply:
관련 커밋입니다: https://github.com/woowacourse/html-apple-store/pull/19/commits/e7dc93e2a522dd1c4a4318030c82ab0cb19943ad